src/components/menubar/index.tsx

```diff
@@ -33,15 +34,24 @@ function Menubar(props: any) {
       <Menu.Item className="menu-item">
         <span>About</span>
       </Menu.Item>
+      {loggedIn &&   <Menu.Item className="menu-item">
+          <span>FAQs</span>
+        </Menu.Item>
+      }
       <Menu.Item className="menu-item">
         <span>Resources</span>
       </Menu.Item>
-      <Menu.Item className="menu-item">
-        <span>Workspace</span>
-      </Menu.Item>
-      <Menu.Item className="menu-profile menu-item-no">
-        <img src={daniel} alt="avatar" />
-      </Menu.Item>
+      {loggedIn &&   <Menu.Item className="menu-item">
+          <span>Workspace</span>
+        </Menu.Item>
+      }
+      {loggedIn &&
+        <Menu.Item className="menu-profile menu-item-no">
+          <img src={daniel} alt="avatar" />
+        </Menu.Item>}
+      {!loggedIn && <Menu.Item className="menu-item login-tab">
+        <span>Login</span>
+      </Menu.Item>}
```

Comment:
Why not use an if else operator here if you're comparing if someone is logged in? ![image](https://i.imgur.com/4am09LS.png)